src/main/java/uk/gov/pay/ledger/report/dao/ReportDao.java

```diff
@@ -66,6 +71,24 @@ public TransactionsStatisticsResult getTransactionSummaryStatistics(TransactionS
         });
     }
 
+    public List<TimeseriesReportSlice> getTransactionsVolumeByTimeseries(ZonedDateTime fromDate, ZonedDateTime toDate) {
+        return jdbi.withHandle(handle -> handle.createQuery("SELECT " +
+                    "date_trunc('hour', t.created_date) as timestamp, " +
+                    "COUNT(1) as all_payments, " +
```

Comment:
we `count(*)` not be used instead? it has better performance than `count(1)`

Reply:
Curious, this (quite in depth) analysis agrees with that but we've had issues with `COUNT(*)` before -- I wonder if it was just because indexes were being incorrectly used. Happy to change it!

Reply:
Oh it was the ID itself the code was counting before -- this shows the fairly significant performance improvement https://github.com/alphagov/pay-ledger/pull/259

Reply:
i'm not really an expert here, i just made the original comment based on some quick searching. i'm happy to settle for whatever you think is best here!